action-templates/actions/action-pr-labeler/action.yml

```
@@ -0,0 +1,26 @@
+  repository:
+    required: false
+    default: ""
+    type: string
+    description: Path to an repository to reference external configuration file
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout external repository
+      if: ${{ inputs.repository != '' }}
+      uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)